cmds/webboot/webboot_test.go

```diff
@@ -275,3 +275,39 @@ func stringToKeypress(str string) []string {
 	}
 	return keyPresses
 }
+
+func TestMirrors(t *testing.T) {
+	uiEvents := make(chan ui.Event)
+	keyPresses := []string{"0", "<Enter>", "0", "<Enter>"}
+	go pressKey(uiEvents, keyPresses)
+	entry := &Config{label: "Arch"}
+	urlResult, mirrorNameForTestPurposes, err := mirrorMenu(entry, uiEvents, "")
+	if err != nil {
+		t.Fatalf(err)
+	}
+	const testLink = "http://mirrors.acm.wpi.edu/archlinux/iso/2021.06.01/archlinux-2021.06.01-x86_64.iso"
+	if urlResult != testLink {
+		t.Fatalf("Wrong mirror link. The link must be the default one. Expected \"" + testLink + "\" but got \"" + urlResult + "\"")
+	}
+	if mirrorNameForTestPurposes != "Default" {
+		t.Fatalf("The first mirror must be the default mirror and be called \"Default\"")
```

Comment:
The general test message pattern is
thing: got whatever, want whatever.

t.Fatalf("First mirror: got %q, want %q", m, "Default")